rundeckapp/grails-spa/src/pages/community-news/NewsletterSubscribe.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <section>
+    <button
+      @click="isSubscribeModalActive = true"
+      class="btn red-button"
+    >{{$t("message.subscribe")}}</button>
+    <modal v-model="isSubscribeModalActive" ref="modal" :header="false" :footer="false">
+      <div class="modal-body">
+        <div v-if="!showConfirmation">
+          <h4>Receive updates in your Inbox.</h4>
+          <form @submit.prevent="handleSubmit">
+            <div class="form-group">
+              <label class="label">Email*</label>
+              <div class="control">
+                <input
+                  type="email"
+                  v-model="email"
+                  class="form-control"
+                  placeholder="Type Your Email..."
+                >
+              </div>
+            </div>
+            <div class="field" style="margin-top:2em;">
+              <div class="control">
+                <button type="submit" class="btn btn-block btn-lg red-button">Subscribe</button>
+              </div>
+            </div>
+          </form>
+        </div>
+        <div v-else>
+          <p>{{confirmationMessage}}</p>
+          <p class="is-size-6">This modal will close in {{counter}} seconds.</p>
+        </div>
+      </div>
+    </modal>
+  </section>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+  name: "NewsletterSubscribeButton",
+  data() {
+    return {
+      email: "",
+      isSubscribeModalActive: false,
+      confirmationMessage: "Thank you for your submission.",
+      showConfirmation: false,
+      counter: 5
+    };
+  },
+  methods: {
+    handleSubmit() {
+      axios
+        .post("https://api-stage.rundeck.com/spark/v1/subscribe", {
```

Comment:
Eventually it should point to api.rundeck.com/online/v1/subscribe but that path change hasn't been made yet, so I figured this would be fine for now until we get it updated.